docs/extend/theme.md

```diff
@@ -1 +1,26 @@
-# Quick Start
\ No newline at end of file
+# Themes
+
+Flarum "themes" are just extensions. Typically, you'll want to use the `Frontend` extender to register custom LESS and JS.
+Of course, you can use other extenders too: for example, you might want to support settings to allow configuring your theme.
+
+You can indicate that your extension is a theme by setting the "extra.flarum-extension.category" key to "theme". For example:
+
+```json
+{
+    // other fields
+    "extra": {
+        "flarum-extension": {
+        "category": "theme",
+    // other fields
+}
+```
```

Comment:
Not sure if the comments render right with JSON, but JSONC (JSON w/ comments) should definitely be ok.

We can also [highlight a specific line using curly braces](https://vuepressbook.com/tutorial/tutorial6.html#code-listings-in-markdown-files).

```suggestion
```jsonc{4}
{
    // other fields
    "extra": {
        "flarum-extension": {
            "category": "theme",
        }
    }
    // other fields
}
```
```